ci/docker-compose.prod.yaml

```diff
@@ -37,8 +37,44 @@ services:
     image: gcr.io/google.com/gchips-productivity/resultstoresearch-server
     environment: 
       GOOGLE_APPLICATION_CREDENTIALS: /root/keys/keyfile.json
+      DATABASE_HOST: db
+      DATABASE_USERNAME: postgres
+      DATABASE_PASSWORD: postgress
```

Comment:
those creds shouldn't be checked into github?

Reply:
Definitely make sure these credentials aren't placed here.